x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { memo, useEffect } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import type {
+  NewPackagePolicy,
+  PackagePolicyCreateExtensionComponentProps,
+} from '@kbn/fleet-plugin/public';
+import type { PostureInput } from '../../../common/constants';
+import {
+  getPolicyWithUpdatedInputs,
+  getPolicyWithHiddenVars,
+  getPolicyWithInputVars,
+  inputsWithVars,
+  getPostureInput,
+  type NewPackagePolicyPostureInput,
+} from './utils';
+import { AwsCredentialsForm } from './aws_credentials_form';
+import { PolicyInputSelector } from './policy_template_input_selector';
+
+const DEFAULT_INPUT_TYPE = {
+  kspm: 'cloudbeat/cis_k8s',
+  cspm: 'cloudbeat/cis_aws',
+} as const;
+
+const DEFAULT_AWS_VARS_GROUP = 'assume_role';
```

Comment:
this seems simple enough without adding any `useEffect` to accomplish this. 

`DEFAULT_AWS_VARS_GROUP` is not exported because it's used only in this file

Reply:
note that i said "use the exported consts" and not "export the consts"
i meant that `cloudbeat/cis_k8s` and `cloudbeat/cis_aws` are already exported from `x-pack/plugins/cloud_security_posture/common/constants.ts` and we should use that instead of creating new strings